dhcpv4/dhcpv4.go

```diff
@@ -557,8 +572,15 @@ func (d *DHCPv4) SetOptions(options []Option) {
 }
 
 // AddOption appends an option to the existing ones.
+// If the last option is an OptionEnd, it will be inserted before that.
```

Comment:
Thanks for updating the comment. What I meant here is to state explicitly that this doesn't deal with End options that appear before the end, like in malformed packets. Not stating that may be implicitly misleading